competitions/template/etl/compose-and-upload

```diff
@@ -0,0 +1,269 @@
+                "10 to 25 Full-time Employees",
+                "26 to 50 Full-time Employees",
+                "51 to 100 Full-time Employees",
+                "101 to 300 Full-time Employees",
+                "301 to 500 Full-time Employees",
+                "501 to 1,000 Full-time Employees",
+                "1,000+ Full-time Employees",
+            ],
+        )
+    )
+
+    comp.add_toc(
+        toc.GeographicToc(
+            "Current_Work_Locations",
+            [
+                [
+                    "Location of Current Work #1 Country",
+                    "Location of Current Work #1 State /Province",
+                ],
+                [
+                    "Location of Current Work #2 Country",
+                    "Location of Current Work #2 State /Province",
+                ],
+                [
+                    "Location of Current Work #3 Country",
+                    "Location of Current Work #3 State /Province",
+                ],
+                [
+                    "Location of Current Work #4 Country",
+                    "Location of Current Work #4 State /Province",
+                ],
+                [
+                    "Location of Current Work #5 Country",
+                    "Location of Current Work #5 State /Province",
+                ],
+            ],
+        )
+    )
+
+    comp.add_toc(
+        toc.GeographicToc(
+            "Future_Work_Locations",
+            [
+                [
+                    "Location of Future Work #1 Country",
+                    "Location of Future Work #1 State / Province",
+                ],
+                [
+                    "Location of Future Work #2 Country",
+                    "Location of Future Work #2 State /Province",
+                ],
+                [
+                    "Location of Future Work #3 Country",
+                    "Location of Future Work #3 State /Province",
+                ],
+                [
+                    "Location of Future Work #4 Country",
+                    "Location of Future Work #4 State /Province",
+                ],
+                [
+                    "Location of Future Work #5 Country",
+                    "Location of Future Work #5 State /Province",
+                ],
+            ],
+        )
+    )
+
+    comp.process_tocs()
+
+    if tdc_config_dir is not None:
+        tdc.AllProposals(comp).generate(tdc_config_dir)
+        tdc.ValidProposals(comp, "Admin Review Status", "Valid").generate(
+            tdc_config_dir
+        )
+        tdc.AllColumns(comp).generate(tdc_config_dir)
+        tdc.ProcessedSpreadsheet(comp).generate(tdc_config_dir)
+
+    my_wiki = wiki.WikiSession(
+        config.username, config.password, comp.name, config.wiki_url
+    )
+    my_wiki.csv_only = csv_only
+    my_wiki.upload_sheet(comp)
+    my_wiki.upload_attachments(attachments.attachments)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
FYI I didn't review the code itself.  Wish I could; just time constraints.  I know it is just based on already-working code, so that's why I felt semi-okay skipping a code review.